src/components/ose/EditQuota.vue

```diff
@@ -3,18 +3,18 @@
         <div class="hero is-light">
             <div class="hero-body">
                 <div class="container">
-                    <h1 class="title"><i class="material-icons">cached</i> Projekt Quotas anpassen</h1>
+                    <h1 class="title"><i class="material-icons">cached</i>Adjust project Quotas </h1>
                 </div>
                 <h2 class="subtitle">
-                    Hier kannst du CPU / Memory Quotas deines Projektes anpassen. Alle Anpassungen werden geloggt.</h2>
+                    you can adjust the CPU / memory quotas here. All changes will be logged!</h2>
             </div>
         </div>
         <br>
         <form v-on:submit.prevent="editQuotas">
             <cluster-select v-model="clusterid"></cluster-select>
             <project-select v-bind:clusterid="clusterid" v-bind:project.sync="project"></project-select>
 
-            <b-field label="Neue CPU Quotas [Cores]"
+            <b-field label="New CPU quotas [Cores]"
```

Comment:
```suggestion
            <b-field label="New CPU quota [cores]"
```
Es gibt nur ein CPU quota. Also kein Plural.